qiskit/assembler/assemble_schedules.py

```diff
@@ -93,6 +93,9 @@ def _assemble_experiments(
                                     converters.InstructionToQobjConverter)
     instruction_converter = instruction_converter(qobj.PulseQobjInstruction,
                                                   **run_config.to_dict())
+
+    schedules = [sched if isinstance(sched, pulse.Schedule) else pulse.Schedule(sched)
+                 for sched in schedules]
```

Comment:
Isn't this because of https://github.com/Qiskit/qiskit-terra/issues/4523? Could we put this in a separate PR and merge it quickly in this release?

Reply:
Oh you're right. Didn't realize this was a separate bug.

Reply:
Added PR #4870